### DISCUSSION TITLE
request to be able to make QR code photoboard for tags

### DISCUSSION THREAD

**Discussion by lbestock** [arch-kiosk/arch-kiosk-office#1119] [General]
Not needed until there is an actual field season! But this would be helpful. The context of the request: 
I am writing about Unit FI, in which we cut a sondage. By definition a sondage has multiple loci in it. My most useful tags are sondage tags and I am using them all the time in analysis. Currently I was just pulling my hair out that I do not have a good final photograph to publish of FI Sondage 1 that shows the stratigraphy, which would illustrate a point in the article. But in fact I do have such photographs, with QR code boards in no less, it's just that they're dayplans, so they don't pop up in tag analysis, because

oh wait, this is a bigger issue

For tags to get QR codes they would have to be contexts themselves. So the answer here would have been to go in and use the gallery to put the image in at least one of the relevant loci? 

This started as a request and has turned into a needs discussion. Putting Sam on it too because it's not clear if this will or should get a database solution or an instructions to users solution.

**Comment by urapadmin** [comment]
that is not possible. Tags don't store images. So I would not know where to put the image. You either would have to make Sondages something else than a tag (a new type of unit?!?) or we wait until we have a somewhat more flexible tag system that can include more than just loci. But the image would still be bound to either a unit or a locus and that is what the qr-code addresses.

**Comment by lbestock** [comment]
The user way to get around this, which was sometimes but not always done in the last Uronarti season (before this was thought about and thus must qualify as basically intuitive or at least teachable), would be to use the QR board for A PARTICULAR locus, that itself is tagged as belonging to the sondage (or whatever). And/or to add a description and/or context to the photo after import if no QR code is used. I tend not to remember if I should be searching tag or description in the file repository and thus usually search them both to find what I am looking for.

**Reply by urapadmin** [reply]
But the photo shows a locus, doesn't it? QR are in the photos of the things we photograph. Do we photograpg a Sondage? And if so what is it? Shouldn't tags then have photos (which would really lift them to a different level of existence)?
I have the feeling making the photo of the locus is simply the right thing to do and not something to go around this.

**Comment by lbestock** [comment]
Well, the photo includes the locus, but it is not of the locus. It includes rather a part of the locus - the part sliced by the sondage. I definitely photographed the sondages, and did so long before I tagged them even. Sometimes they ended up in dayplans and sometimes in loci and, while it is more accurate to put them in dayplans (they include multiple loci) it was more useful to put them in loci (because then I retrieved them with tag analysis). Should tags have photos? I am not totally sure. A sondage tag would definitely benefit from it - the sondage is a physical entity that is not any other entity and gets photographed, and repeatedly. But a tag that is a phase? There is no way to photograph that. Hm, but I can imagine a map that has all phase 1 architecture shown and nothing else, and that would in effect be an image of phase 1. Not photographable, but still imageable and in fact very important to make an image of it. And even straightforward with the tags, if we had GIS linked and you could easily produce a map that showed only those loci tagged "phase 1". And THAT is an image we would publish, an image we spend a great deal of time making in order to publish. So perhaps you are right. The locus photo as a workaround was a bit different. If I photograph all of locus 3 to describe locus 3 that is one thing. If I photograph sondage 1, which includes locus 3 sandwiched between loci 1 and 4, then that is different. It is wrong to call it locus 3 in the sense that the photo doesn't show all of locus 3, isn't primarily about locus 3. But it would be wrong NOT to attach it to locus 3, too, since it illustrates something critical about locus 3. And it isn't the case that we expect all locus photos to show the whole locus - an opening photo of a locus necessarily captures it at the moment it is defined, long before we can know how far it extends or whatnot. So I guess (I am so talking to myself here, while you torture the bike upstairs) that that photo of the sondage does belong in the locus. But it belongs in EVERY locus that is pictured in it, and that gets insane and doesn't happen and in part needn't happen because, once those loci are all tagged "sondage 1", putting it in every individual locus would be a huge waste of time.
